twinkle-ansi/src/main/java/org/codejive/twinkle/ansi/Ansi.java

```diff
@@ -100,19 +109,23 @@ public static String backgroundBright(int index) {
     }
 
     public static String foregroundIndexed(int index) {
-        return FOREGROUND_COLORS + ":" + COLORS_INDEXED + ":" + index;
+        // Standard SGR: 38;5;<n>
```

Comment:
not sure who is right here - it does not seem to matter but online i find ; being used not : - which one is correct/expected?